README.md

```diff
@@ -1,70 +1,156 @@
+### Install
 
-### Analyzing the Bundle Size
+Install this project with the following commmands in the terminal:
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+ ```sh
+npm init -y
+```
+` cd your-project-folder`
 
-### Making a Progressive Web App
+```sh
+npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+```
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+```sh
+npm install --save-dev hint@7.x
+```
 
-### Advanced Configuration
+```sh
+npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
+```
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+### Usage
 
-### Deployment
+To run the project,
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+Open the browser and enter the adress of index.html file from your computer/server
 
-### `npm run build` fails to minify
+### Run tests
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+- For HTML
+```sh
+npx hint .
+```
+- For CSS
+```sh
+npx stylelint "**/*.{css,scss}"
+```
+
+-For JavaScript
+```sh
+npx eslint .
+```
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Surafel Samson**
+
+- GitHub: [@Surafels](https://github.com/Surafels)
+- Twitter: [@SurafelSamson2](https://twitter.com/SurafelSamson2)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Dark mode**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+## 🤝 Contributing <a name="contributing"></a>
+
+You are welcome, and click <a href="https://github.com/Surafels/Math-Magicians/issues">here</a> for any issues🙏
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please give it a star and follow me on github linkedIn and Twitter
+, you are also welcome for any suggestions🙏
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse. 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] the link to your MIT license is not working as expected, please have another look at the template provided by Microverse and adjust it accordingly.